test/runnable/cpp_abi_tests.d

```diff
@@ -1,4 +1,11 @@
 // EXTRA_CPP_SOURCES: cpp_abi_tests.cpp
+// CXXFLAGS: -std=c++11
```

Comment:
This looks like it will need to be handled by a makefile. The Windows boxen don't like it.

Reply:
You can make the option depend on the OS variable:
```
// CXXFLAGS(linux freebsd osx netbsd dragonflybsd): -std=c++11
```

Reply:
Yeah but windows needs to be tested with c++11+ as well.

Reply:
@thewilsonator MSVC doesn't have a C++11 switch, but it defaults to the latest fully-supported standard

Reply:
Ah, good to know.

Reply:
cl from VS2019 shows this in the command line help:
```
/std:<c++14|c++17|c++latest> C++ standard version
    c++14 – ISO/IEC 14882:2014 (default)
    c++17 – ISO/IEC 14882:2017
    c++latest – latest draft standard (feature set subject to change)```

Reply:
As long as the default is >c++11 it doesn't really matter. What is very annoying is that VS2013 doesn't support char16_t/char32_t, and the autotester machines seem to use that.